src/backend/app/modules/Apps/CustomerRegistration/CustomerRegistrationAppService.php

```diff
@@ -55,8 +55,9 @@ public function createCustomer(AndroidAppRequest $request): Person {
             'in_use' => 1,
         ];
         $meter = $this->meterService->create($meterData);
+        $personData = $person->toArray();
```

Comment:
I assume you do this to resolve

```
Access to an undefined property Illuminate\Database\Eloquent\Model::$id.
```

Whoever, here you are not really fixing the root cause of the issue, but the symptom. The underlying issue is in `PersonService.php`. 

```
public function createFromRequest(Request $request): Model
```

returns a generic `Model`. But should return `Person` instead.